### ISSUE TITLE
broken WYSIWYG items after updating to 9.25.1

### ISSUE THREAD

**Issue by drennvinn** [directus/directus#18236]
### Describe the Bug

I just updated directus from 9.23.4 to 9.25.1 with `npm update` and `npx directus database migrate:latest`
after that, everything seemed to work except WYSIWYG. They all have a problem with some toolbar items.
here is an example of what I get: (it still works when I click on the broken items)
![Capture d’écran 2023-04-18 à 19 24 15](https://user-images.githubusercontent.com/18407632/232855972-4e054f82-3d65-45e2-92cf-19630a00d50e.png)
I tried with a new test collection and fresh WYSIWYG field but i got the same.
I checked the console and there is no error.
I specify that I have no custom code for this project.

### To Reproduce

I created a new directus test project and unfortunately for me it works 😅
it's only after the update. 
I tried the same update operation on my staging server and here too i got the same WYSIWYG problem

### Directus Version

9.25.1

### Hosting Strategy

Self-Hosted (Custom)

**Comment by hanneskuettner** [comment]
This is a duplicate of #18184 and was fixed in #18185

**Comment by drennvinn** [comment]
Thanks!!  That's too weird, I had looked at it though 🤔
